src/main/java/no/unit/nva/model/contexttypes/Book.java

```diff
@@ -1,21 +1,16 @@
 package no.unit.nva.model.contexttypes;
 
-import com.fasterxml.jackson.annotation.JsonSetter;
 import com.fasterxml.jackson.annotation.JsonTypeInfo;
 import no.unit.nva.model.Level;
-import no.unit.nva.model.NullSeries;
-import no.unit.nva.model.Series;
 
 import java.util.List;
 import java.util.Objects;
 
-import static java.util.Objects.nonNull;
-
 @JsonTypeInfo(use = JsonTypeInfo.Id.NAME, property = "type")
 public class Book implements PublicationContext {
 
-    private String title;
-    private Series series;
+    private String seriesTitle;
```

Comment:
Book has no title? or is a book considered an series of books with only one volume?

Reply:
Book has a title, but this is in entity description, here we are talking about the container format.